src/server/types.ts

```diff
@@ -97,12 +97,15 @@ export const mediaTypes: Set<MediaType> = new Set(['screen', 'print']);
 export type ColorScheme = 'dark' | 'light' | 'no-preference';
 export const colorSchemes: Set<ColorScheme> = new Set(['dark', 'light', 'no-preference']);
 
+type browserType = 'chromium' | 'firefox' | 'webkit';
```

Comment:
Please inline this type. We now prefer to inline all types we do not export, for the easier use.